war/src/main/java/org/exoplatform/addons/es/EmbeddedESStartupServlet.java

```diff
@@ -42,27 +46,37 @@ public void init() throws ServletException {
     }
 
     LOG.info("Initializing elasticsearch Node '" + getServletName() + "'");
-    Settings.Builder settings = Settings.settingsBuilder();
+    Settings.Builder settings = Settings.builder();
 
     InputStream resourceAsStream = getServletContext().getResourceAsStream("/WEB-INF/elasticsearch.yml");
-    if (resourceAsStream != null) {
+    if (resourceAsStream == null) {
+      throw new ServletException("Error while initializing elasticsearch node, configuration file '/WEB-INF/elasticsearch.yml' couldn't be found");
+    }
+    try {
       settings.loadFromStream("/WEB-INF/elasticsearch.yml", resourceAsStream);
-      try {
-        resourceAsStream.close();
-      } catch (IOException e) {
-        // ignore
-      }
+      resourceAsStream.close();
+    } catch (IOException e) {
+      throw new ServletException("Error while initializing elasticsearch node '" + getServletName() + "'", e);
     }
 
-    if (settings.get("http.enabled") == null) {
-      settings.put("http.enabled", false);
+    String pathData = settings.get("path.data");
+    if (pathData == null) {
+      settings.put("path.data", System.getProperty("exo.data.dir"));
+    } else if (pathData.startsWith("${")) {
```

Comment:
the es.*** properties aren't taken into account anymore. The ES properties couldn't be injected by System properties anymore, it's injected via System Environment variables only when launching  ES using command line on standalone mode.
So the `./elasticsearch -Des.path.data=XYZ` is now `./elasticsearch -Ees.path.data=XYZ`
So I had to make a change in order to remake the exo.properties variables taken into account. See this:
https://github.com/exo-addons/exo-es-embedded/pull/9/files#diff-3a01cbd89b06b166d1ae0267f0fbf0a5R59